pkg/metrics/sketch_series_test.go

```diff
@@ -94,12 +95,28 @@ func TestSketchSeriesListMarshal(t *testing.T) {
 		require.Len(t, pb.Dogsketches, len(in.Points))
 		for j, pointPb := range pb.Dogsketches {
 
-			check(in.Points[j], pointPb)
+			check(t, in.Points[j], pointPb)
 			// require.Equal(t, pointIn.Ts, pointPb.Ts)
 			// require.Equal(t, pointIn.Ts, pointPb.Ts)
 
 			// fmt.Printf("%#v %#v\n", pin, s)
 		}
 	}
+}
+
+func TestSketchSeriesListJSONMarshal(t *testing.T) {
+	sl := make(SketchSeriesList, 2)
+
+	for i := range sl {
+		sl[i] = makeseries(i)
+	}
+
+	json, err := sl.MarshalJSON()
+	assert.NoError(t, err)
+	assert.Equal(t, string(json), `{"sketches":[{"metric":"name.0","tags":["a:0","b:0"],"host":"host.0","interval":0,"points":[{"sketch":{"summary":{"Min":0,"Max":0,"Sum":0,"Avg":0,"Cnt":0}},"ts":0},{"sketch":{"summary":{"Min":0,"Max":0,"Sum":0,"Avg":0,"Cnt":1}},"ts":10},{"sketch":{"summary":{"Min":0,"Max":1,"Sum":1,"Avg":0.5,"Cnt":2}},"ts":20},{"sketch":{"summary":{"Min":0,"Max":2,"Sum":3,"Avg":1,"Cnt":3}},"ts":30},{"sketch":{"summary":{"Min":0,"Max":3,"Sum":6,"Avg":1.5,"Cnt":4}},"ts":40}]},{"metric":"name.1","tags":["a:1","b:1"],"host":"host.1","interval":1,"points":[{"sketch":{"summary":{"Min":0,"Max":0,"Sum":0,"Avg":0,"Cnt":0}},"ts":0},{"sketch":{"summary":{"Min":0,"Max":0,"Sum":0,"Avg":0,"Cnt":1}},"ts":10},{"sketch":{"summary":{"Min":0,"Max":1,"Sum":1,"Avg":0.5,"Cnt":2}},"ts":20},{"sketch":{"summary":{"Min":0,"Max":2,"Sum":3,"Avg":1,"Cnt":3}},"ts":30},{"sketch":{"summary":{"Min":0,"Max":3,"Sum":6,"Avg":1.5,"Cnt":4}},"ts":40},{"sketch":{"summary":{"Min":0,"Max":4,"Sum":10,"Avg":2,"Cnt":5}},"ts":50}]}]}`+"\n")
```

Comment:
Any reason not to use `JSONEq` here? https://godoc.org/github.com/stretchr/testify/assert#JSONEq

Reply:
Indeed more suited 👍 Done.